File: api/dotNet/FinanceApi/FinanceApi/Repositories/EF Models/vPaymentType.cs
using System.ComponentModel.DataAnnotations;

namespace FinanceApi.Repositories.EF_Models
{
    public class vPaymentType
    {
        public vPaymentType()
        {
            PaymentTypeID = 0;
            PaymentTypeCategoryID = 0;
            PaymentTypeName = string.Empty;
            PaymentTypeDescription = string.Empty;
            PaymentTypeCategoryDescription = string.Empty;
        }
        [Key]
        public int PaymentTypeID { get; set; }
        public string PaymentTypeName { get; set; }
        public string PaymentTypeDescription { get; set; }
        public int PaymentTypeCategoryID { get; set; }
        public string PaymentTypeCategoryDescription { get; set; }
    }
}

File: api/Node/finance-api/src/entity/v_payment_type.ts
import { ViewEntity, ViewColumn} from "typeorm";

@ViewEntity({
  name: "v_payment_type",
  expression: "",
  synchronize: false,
})
export class v_payment_type {
  @ViewColumn({ name: "payment_type_id" })
  paymentTypeID?: number;

  @ViewColumn({ name: "payment_type_category_id" })
  paymentTypeCategoryID?: number;

  @ViewColumn({ name: "payment_type_name" })
  paymentTypeName?: string;

  @ViewColumn({ name: "payment_type_description" })
  paymentTypeDescription?: string;

  @ViewColumn({ name: "payment_type_category_name" })
  paymentTypeCategoryName?: string;

  // @ViewColumn()
  // LastUpdated?: Date;
}
